configuration_version.go

```diff
@@ -180,6 +193,27 @@ func (s *configurationVersions) Read(ctx context.Context, cvID string) (*Configu
 	return cv, nil
 }
 
+// Read ingress attributes by configuration version ID.
+func (s *configurationVersions) ReadIngressAttributes(ctx context.Context, cvID string) (*IngressAttributes, error) {
```

Comment:
Can you add a unit test to `configuration_version_test.go` please? See TestConfigurationVersionsRead.

Reply:
Decided that it's best to add them later.